### PR TITLE
Added etcd username and userpassword from config

### PR DESCRIPTION
**Pull request by ymhhh**
To use etcd with user & password when it should auth

### CODE REVIEW

**Comment by pstibrany** [comment]
Thank you for your PR. To make it into mergeable state, please do the following:
- add two new fields into `RegisterFlagsWithPrefix` method in etcd.go
- after doing that, run `make BUILD_IN_CONTAINER=false doc` to regenerate documentation
- add entry to CHANGELOG file

Thank you!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## master / unreleased
 
+* [ENHANCEMENT] Added etcd username and userpassword from config #4205
```
Comment (pstibrany) [inline CHANGELOG.md:5]: Order of entries in the changelog should be: `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.

```suggestion
* [ENHANCEMENT] Etcd: Added username and password to etcd config. #4205
```

**Review by pstibrany — APPROVED** [review]
LGTM, thank you.

File: CHANGELOG.md (hunk 1 above)
Comment (pracucci) [reply CHANGELOG.md:5]: @ymhhh Could you fix this, please? We would be very happy to merge once done! Thanks!

**Review by pracucci — APPROVED** [review]

**Pull request merged**